app/src/ui/dispatcher/dispatcher.ts

```diff
@@ -1259,7 +1263,7 @@ export class Dispatcher {
       manualResolutions
     )
 
-    if (result === RebaseResult.CompletedWithoutError) {
+    if (result === RebaseResult.ConflictsEncountered) {
```

Comment:
I ran into this which unless I'm missing something was a bug? 😅

Reply:
I believe this is just confusing 😄 

Flow of a rebase flow with conflicts.
1. `git rebase`
2. Interpret result -> if `RebaseResult.ConflictsEncountered`, we show the conflicts dialog.
3. User resolves conflicts
4. User hits "Continue Rebase", this method of `continueRebase` is called and the `result` on line 1262 is the result of `git rebase --continue`. 
    - If `result === RebaseResult.ConflictsEncountered` -> repeat 2-4 (could happen on any number of commits in the operation)
    -   If `result === RebaseResult.CompletedWithoutError` the, the rebase concluded and we want to record an successful operation.
    
 Therefore, it is correct as it was and what you changed it to will lead to incorrectly recording a successful rebase operation with conflicts for every conflict that happens during that rebase operation.
 
It may be easier to see what I am talking about if you set your self up two branches and alternate between adding conflicting changes to a file across a few commits and put a debugger here to watch how many times this is hit during a rebase operation.

That said... If this tripped you up and I can easily see why and remember it being confusing when I refactored this stuff.. Maybe we should put some documentation here explaining it.

Reply:
Ohhh thank you for the context! 💖  So the fact that `continueRebase` is invoked means _there were_ conflicts, and now checking for `CompletedWithoutError` means the rebase finished and there aren't more conflicts.

If that makes sense I will write a comment for future readers 😂

Reply:
Yes! Much more succinct way of saying that. :)